source/code-snippets/updateOne.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func main() {
+	// Replace the uri string with your MongoDB deployment's connection string.
+	uri := "mongodb+srv://<username>:<password>@<cluster-address>/test?w=majority"
+
+	client, err := mongo.Connect(context.TODO(), options.Client().ApplyURI(uri))
+
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		if err = client.Disconnect(context.TODO()); err != nil {
+			panic(err)
+		}
+	}()
+
+	// begin updateone
+	myCollection := client.Database("sample_restaurants").Collection("restaurants")
+	id, _ := primitive.ObjectIDFromHex("5eb3d668b31de5d588f42a7a")
+	filter := bson.D{{"_id", id}}
+	update := bson.D{{"$set", bson.D{{"avg_rating", 4.4}}}}
+
+	result, err := myCollection.UpdateOne(context.TODO(), filter, update)
+	// end updateone
+
+	if err != nil {
+		log.Fatal(err)
```

Comment:
nit: The other two error handling blocks use `panic(err)` instead. Consider always using the same exit call when handling errors (preferably `panic(err)` because it's simple).

Reply:
accepted

Reply:
accepted